### DISCUSSION TITLE
I can't get my repository to show

### DISCUSSION THREAD

**Discussion by Aeppers1** [community/community#40274] [New to GitHub]
Hi I am very new to this and I cannot see any of the items or metadata that I have uploaded on the actual published site. Is there anyone who has experienced something similar or could help me work through this?

**Comment by ossanna16** [comment]
Hi there @Aeppers1 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by AlefMemTav** [comment]
Try see in commits.

**Reply by Aeppers1** [reply]
I'm sorry, what?

**Reply by Aeppers1** [reply]
I see what you're saying. Thank you. I read that as "comments" and not "commits"

**Comment by jfullstackdev** [comment]
you only have one public repo,

![image](https://user-images.githubusercontent.com/47092464/204700047-dbf58623-9561-4d8c-a7d4-c60619fe77a2.png)

and using the free account, if you want to host in GitHub Pages, your repo must be public

that App was generated from

![image](https://user-images.githubusercontent.com/47092464/204700116-01fb9633-ccbf-4181-89c0-b33f2c47717c.png)

and this is the rendered web page for that

![image](https://user-images.githubusercontent.com/47092464/204700208-fb28b234-fef9-4db2-a3be-0cb2eb2e0d36.png)

seems to be working fine, what's the problem ?

**Reply by Aeppers1** [reply]
I've got it now. Thanks.
